changelog.md

```
@@ -8,62 +8,120 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+
+### Changed
+
+- Moved to 2201.12.0 distribution
+
 ## [1.0.0] - 2024-07-16
 
 - Added Initial client implementation
 
 ## sap.s4hana.api_salesorganization_srv
 
+## [Unreleased]
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)